info/kubelessInfo.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const _ = require('lodash');
+const Api = require('kubernetes-client');
+const BbPromise = require('bluebird');
+const chalk = require('chalk');
+const helpers = require('../lib/helpers');
+
+class KubelessInfo {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options || {};
+    this.provider = this.serverless.getProvider('google');
+    this.commands = {
+      info: {
+        usage: 'Display information about the current functions',
+        lifecycleEvents: [
+          'info',
+        ],
+        options: {
+          verbose: {
+            usage: 'Display metadata',
+            shortcut: 'v',
+          },
+        },
+      },
+    };
+    this.hooks = {
+      'info:info': () => BbPromise.bind(this)
+        .then(this.validate)
+        .then(this.infoFunction),
+    };
+  }
+
+  validate() {
+    helpers.validateEnv();
+    const unsupportedOptions = ['stage', 'region'];
+    helpers.warnUnsupportedOptions(
+      unsupportedOptions,
+      this.options,
+      this.serverless.cli.log.bind(this.serverless.cli)
+    );
+    return BbPromise.resolve();
+  }
+
+  formatMessage(service, functions, options) {
+    if (options && !options.color) chalk.enabled = false;
+    let message = '';
+    message += `\n${chalk.yellow.underline('Service Information')}\n`;
+    message += `${chalk.yellow('Service: ')} ${service.name}\n`;
+    message += `${chalk.yellow('Cluster IP: ')} ${service.ip}\n`;
+    message += `${chalk.yellow('Ports: ')}\n`;
+    _.each(service.ports, (port) => {
+      message += `  ${chalk.yellow('Protocol: ')} ${port.protocol}\n`;
+      message += `  ${chalk.yellow('Port: ')} ${port.port}\n`;
+      message += `  ${chalk.yellow('Target Port: ')} ${port.targetPort}\n`;
+      message += `  ${chalk.yellow('Node Port: ')} ${port.nodePort}\n`;
```

Comment:
in case it is not defined it would print `Node Port: undefined`. I will add a check for that